parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -184,6 +184,9 @@ $global:TLSBootstrapToken = "{{GetTLSBootstrapTokenForKubeConfig}}"
 
 $global:IsNotRebootWindowsNode = [System.Convert]::ToBoolean("{{GetVariable "isNotRebootWindowsNode" }}");
 
+# Disable OutBoundNAT in Azure CNI configuration
+$global:IsDisableWindowsOutboundNat = [System.Convert]::ToBoolean("{{GetVariable "isDisableWindowsOutboundNat" }}");

```

Comment:
Should we set `$WindowsCSEScriptsPackage` to "aks-windows-cse-scripts-v0.0.14.zip" in this PR?

Reply:
Updated. Thank you.